RELEASE_NOTES_LATEST.md

```diff
@@ -29,66 +29,28 @@ Documentation: https://typedb.com/docs/drivers/java/overview
     <dependency>
         <groupid>com.vaticle.typedb</groupid>
```

Comment:
This is out of scope for today, but we should decide if we want to shrink this from
`com.vaticle.typedb` -> `com.typedb`

Reply:
You're right, I'll at least add a task for it!